buildscripts/sonatype-upload.sh

```diff
@@ -87,7 +87,6 @@ REPOID="$(
   grep stagedRepositoryId |
   sed 's/.*<stagedRepositoryId>\(.*\)<\/stagedRepositoryId>.*/\1/'
   )"
-echo "Repository id: $REPOID"
```

Comment:
How is this a secret? It's "iogrpc-" with a predictable incrementing number on the end. Won't we want to know this anyway to issue the release?